### ISSUE TITLE
disable workaround for vertical crop in EPS backend in fullpage mode

### ISSUE THREAD

**Issue by fedelibre** [jperon/lyluatex#222]
Issue #207 has been fixed and the score compiles fine with default `insert=systems`.
But if you compile a score having a StaffGroup and use `insert=fullpage`, SystemStartBracket is no more aligned with the staves.

Can you enable the workaround only when `insert=systems` is used?

You can reproduce the error using:

    \lilypondfile[insert=fullpage]{file.ly}

The file should contain a StaffGroup:

```
\version "2.19.82"

\new StaffGroup 
  <<
  \new Staff { \clef bass g1 }
  \new Staff { \clef bass g1 }
  >>
```

**Comment by fedelibre** [comment]
@jperon I suggest you revert commit a8f75aab95a8c2504871750c9638c4576e27e3f2

While it solves the problem of badly cropped StaffGroup brackets, it causes many other collision problems.
I've reverted the commit to be able to use `insert=fullpage` and when I tried again `insert=systems` I realized that all the collision problems I experienced before disappeared.

BTW, git logs are quite cryptic and without Github are meaningless. If you have to move to another platform one day, you'll regret you've not added a small description of each commit.

**Comment by jperon** [comment]
@fedelibre Sorry for the delay. I reverted the change, and made it optional: if you need the fix to avoid badly cropped StaffGroup brackets, you just have to add the option `fix_badly_cropped_staffgroup_brackets` (disabled by default). If it's enabled globally, `nofix_badly_cropped_staffgroup_brackets` cancels it locally.

**Comment by fedelibre** [comment]
@jperon This problem was fixed in LilyPond 2.23.x so the next stable 2.24 won't need the workaround:
https://gitlab.com/lilypond/lilypond/-/merge_requests/524

**Comment by jperon** [comment]
Thank you for the reference. As `lyluatex` could be used with older versions of LilyPond, and as this fix is opt-in, I’ll keep it as is (at least for now).
